lib/streamlit/config.py

```diff
@@ -1211,8 +1220,45 @@ def func_with_lock():
     return disconnect
 
 
+def _check_static_files_config():
+    # Import logger locally to prevent circular references
+    from streamlit.logger import get_logger
+
+    LOGGER = get_logger(__name__)
+
+    # TODO [KAREN] Rephrase warning messages
+    if get_option("server.enableStaticServing"):
+
+        static_folder_path = os.path.abspath("static")
+
+        if not os.path.isdir(static_folder_path):
+            LOGGER.warning(
+                "STATIC FILE SERVING ENABLED, but folder static/ "
+                "doesn't exist :( :( :("
+            )
+        else:
+            try:
+                real_static_folder_path = file_util.sanitize_user_static_path(
+                    static_folder_path
+                )
+            except RuntimeError as e:
+                LOGGER.error(str(e))
+                set_option(
+                    "server.enableStaticServing",
+                    False,
+                    ConfigOption.STREAMLIT_DEFINITION,
+                )
```

Comment:
I guess this try/except block is not really needed right now since we have a fixed `static_folder_path`?

But in the case that we keep it and get the RuntimeError, I think I prefer to propagate the exception (so the developer will definitely see it and deal with it) rather than swallow it and implicitly turn off the config? WDYT?